test/config.json

```diff
@@ -322,7 +322,9 @@
 		},
 		"vtgatev2": {
 			"File": "vtgatev2_test.py",
-			"Args": [],
+			"Args": [
+				"--vtgate-gateway-flavor=shardgateway"
```

Comment:
I changed the default value of --vtgate-gateway-flavor to discoverygateway, so I added an explicit value here for the one that still tests shardgateway. Internally I checked that we always explicitly specify --vtgate-gateway-flavor in our test configs.